docs/api-gateway-v2/03-04-migration.md

```diff
@@ -44,105 +44,22 @@ Follow these steps:
-
-b. Copy the values for **service.name** and **service.port** parameters from the corresponding fields of the Api CR. 
-
-c. Set the value for **service.host** parameter to any temporary value which includes the domain. For example, if the value for the **hostname** parameter of the Api CR was set to `sample-service.kyma.local`, change it to `temp-sample-service.kyma.local` so that differs from the original value. Make sure other services on your cluster do not use this hostname. 
-
-d. Configure the **rules** parameter of APIRule CR.
-
->**NOTE:** For details on authentication configuration and possible values you can use, see [this](#authentication-of-api-resources-and-apirules) section.
-
-When the configuration is ready, create the APIRule object and test if the service is working as expected on the new host. It should work the same on both hosts.
-
-
-3. Remove the dependent resources of the Api CR:
-
-   ```shell script
-     kubectl delete virtualservice {API_NAME} -n {NAMESPACE}
-    ```
-Make sure that the Virtual Service resource is deleted before proceeding.
-
-4. If the Api CR was secured with an authentication mechanism, delete the Policy resource:
-
-    ```shell script
-     kubectl delete policy {API_NAME} -n {NAMESPACE}
-    ```
-
-5. To make the service returns to its original host, set the **spec.service.host** parameter of APIRule CR to the value used by the Api CR.
-
-    ```shell script
-    kubectl edit apirule {APIRULE_NAME} -n {NAMESPACE}
-    ```
-
-After saving that configuration, the service should be available on the original hostname.
-
-6. Remove the Api CR:
-
-    ```shell script
-    kubectl delete api {API_NAME} -n {NAMESPACE}
-    ```
-
-## Authentication of Apis and APIRules
+2. Create an APIRule CR based on the Api CR's specification.
 
-To properly configure the authentication mechanism for the new APIRule, learn more about the differences in authentication configuration for Api and APIRule CRs:
+>**NOTE:** Do not copy the [**status**](components/api-gateway-v2#custom-resource-api-rule-additional-information) parameter from the original Api CR.
 
-| Api CR       | APIRule CR| 
-|--------------| ----------|
-|Configured with a list of JWT issuers set for all paths of a service.| Configured with a list of path definitions along with their respective authentication methods.|
-| You can define a list of excluded paths for a given issuer. Requests for these paths won't require authentication by this issuer but will require authentication from other configured issuers.| You can specify what authentication to use per path definition and use a regular expression to match every path, or a subset of possible paths, or a single one. |
+| Parameter | Action  |
+|-----------|---------|
+| **metadata.name**| Specify the name of the API Rule.| 
+| **spec.gateway**| Replace the default value (`kyma-gateway.kyma-system.svc.cluster.local`) with Istio Gateway used to expose services.|
```

Comment:
The user should not replace the default value. Instead, most probably they should use the default value, as it is the default Istio Gateway in Kyma used to expose services. 

There was no field specifying gateway in an Api CR, as it was set by default to the value `kyma-gateway.kyma-system.svc.cluster.local`. In APIRule the user can use a different Isito Gateway, but it is not something we suggest.